### ISSUE TITLE
Selecting an empty field turns null into empty string

### ISSUE THREAD

**Issue by lefleets** [realm/realm-studio#1159]
In Realm Studio it is possible to select an empty (`null`) field for editing. However once you do, and blur / leave that field the value is no longer `null`, but an empty `string?` in my case. I would rather leave everything as it is - so how do I replace the null value?

I think it makes more sense to convert an empty string `''` to null when deselecting the field, because the only way this value could have entered that field is by using Realm Studio to work in the db directly - not from any application.

Or, perhaps a menu option to place the `null` value in the field, if it stays as a string with 0 characters after you deselect the field.

To put it another way, selecting a field for editing commits you to making a change before you've really added any changes.
